diff -uraN miniupnpc-2.2.3/CMakeLists.txt omv-miniupnpc-2.2.3/CMakeLists.txt
--- miniupnpc-2.2.3/CMakeLists.txt	2021-09-28 23:41:25.000000000 +0200
+++ omv-miniupnpc-2.2.3/CMakeLists.txt	2021-10-12 15:22:14.489230967 +0200
@@ -141,7 +141,6 @@
   add_library (miniupnpc::miniupnpc ALIAS libminiupnpc-shared)
   set_target_properties (libminiupnpc-shared PROPERTIES EXPORT_NAME miniupnpc)
   set_target_properties (libminiupnpc-shared PROPERTIES OUTPUT_NAME "miniupnpc")
-  set_target_properties (libminiupnpc-shared PROPERTIES VERSION ${MINIUPNPC_VERSION})
   set_target_properties (libminiupnpc-shared PROPERTIES SOVERSION ${MINIUPNPC_API_VERSION})
   target_link_libraries (libminiupnpc-shared PRIVATE miniupnpc-private)
   target_compile_definitions(libminiupnpc-shared PRIVATE MINIUPNP_EXPORTS)
